RecoTracker/FinalTrackSelectors/python/preDuplicateMergingGeneralTracks_cfi.py

```diff
@@ -18,7 +18,7 @@
         cms.InputTag("muonSeededTracksInOutSelector","MVAVals"), 
         cms.InputTag("muonSeededTracksOutInSelector","MVAVals"), 
     ),
-    setsToMerge = cms.VPSet(cms.PSet(pQual = cms.bool(False), tLists = cms.vint32(0, 1,2))),
+    setsToMerge = cms.VPSet(cms.PSet(pQual = cms.bool(True), tLists = cms.vint32(0, 1,2))),
```

Comment:
if this removes the discrepancies, let's go for it then.
Otherwise, more work is needed to understand why some of the tracks change.